shell/platform/android/io/flutter/embedding/android/AndroidTouchProcessor.java

```diff
@@ -59,15 +61,15 @@ public class AndroidTouchProcessor {
 
   // Must match the PointerSignalKind enum in pointer.dart.
   @IntDef({PointerSignalKind.NONE, PointerSignalKind.SCROLL, PointerSignalKind.UNKNOWN})
-  private @interface PointerSignalKind {
+  public @interface PointerSignalKind {
     int NONE = 0;
     int SCROLL = 1;
     int UNKNOWN = 2;
   }
 
   // Must match the unpacking code in hooks.dart.
   private static final int POINTER_DATA_FIELD_COUNT = 35;
-  private static final int BYTES_PER_FIELD = 8;
+  public static final int BYTES_PER_FIELD = 8;
```

Comment:
can this be package only? also, it's missing the annotation `@VisibleForTesting`.